docs/contributing/testing/database/index.md

```diff
@@ -0,0 +1,92 @@
+# Database Integration Testing
+
+Since Bitwarden has multiple database options (4), testing them all automatically is incredibly
```

Comment:
🎨 I'd leave "4" out of this, as it could potentially change and this would be an easy miss.